fdbclient/VersionVector.h

```diff
@@ -88,17 +94,12 @@ struct VersionVector {
 			return; // rerurn an invalid version vector
 		}
 
-		std::map<Version, std::set<Tag>> tmpVersionMap; // order versions
```

Comment:
We will need to make a similar change in the "apply" method too?

Reply:
Done.